src/cirrusTheme.ts

```diff
@@ -56,15 +56,6 @@ let cirrusTheme = {
       hover: cirrusColors.cirrusGrey,
     },
   },
```

Comment:
Just a note for reviewers - this override was removed so I have to do it in the CSS